App.dart

```diff
@@ -0,0 +1,82 @@
+import 'dart:io';
+import 'Home.dart';
+import 'Login.dart';
+import 'SignUp.dart';
+import 'Welcome.dart';
+
+// Coordinator
+class App {
+  late Welcome welcome = Welcome();
+  late Login login = Login();
+  late SignUp signUp = new SignUp();
+  late Home home = new Home();
+
+  void runHome(UserOptions x) {

```

Comment:
the function name says something and does totally something else.
the function says 'run home' but I don't see any statement that says run home.